### PR TITLE
WEB-672: Implementing Guide Pagination

### PR DESCRIPTION
**Pull request by Finholt**

### CODE REVIEW

File: modules/pages.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ const Pages = {
   },
 
   under(path: string): Page[] {
-    const normalizedPath = `/${path.replace(/^\//, "").replace(/\/$/, "")}/`;
+    const normalizedPath = `/${path.replace(/^\//, "").replace(/\/$/, "")}`;
```
Comment (Finholt) [inline modules/pages.ts:45]: Removed trailing `/` in order to return the index file under the path as well

File: next.config.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ const { PHASE_DEVELOPMENT_SERVER } = require("next/constants");
 module.exports = (phase) =>
   withImages(
     withMdxEnhanced({
-      // layoutPath: 'layouts',
-      // defaultLayout: true,
+      layoutPath: "layouts",
+      defaultLayout: true,
```
Comment (Finholt) [inline next.config.js:10]: Applies `layouts/index.tsx` by default if no layout is specified in the frontmatter

File: layouts/guides.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+              nextHref={nextPage ? nextPage.id : ""}
+            />
+          </StyledPagination>
+        )}
+
+        <FooterCTA
+          topic="Test in the Sandbox for free today."
+          description="Use sandbox environment to test API requests."
+          button={{ text: "Get API Keys" }}
+        />
```
Comment (Finholt) [inline layouts/guides.tsx:47]: Using these layouts, we gain access to the frontMatter and can add content across several pages. In this case, I'm adding the pagination and footer cta to all pages with `layout: guides` in their frontmatter

**Review by sausman — APPROVED** [review]

**Pull request merged**